lib/ezfile/classes/ezgzipzlibcompressionhandler.php

```diff
@@ -51,11 +51,15 @@ function compressionLevel()
     static function isAvailable()
     {
         $extensionName = 'zlib';
+
+        // dl() is removed in PHP 7.0, see https://secure.php.net/manual/en/function.dl.php
+        /*
```

Comment:
Please rather remove all commented out code, feel free to clean up surrounding commented out code as well _(ref some of the `set_magic_quotes_runtime ` changes)_

Reply:
Replaced code with `function_exists('gzcompress')`

Also removed commented `set_magic_quotes_runtime` stuff